atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -426,6 +426,7 @@ void deleteAllTimestamps(TableReference tableRef,
             TableReference tableRef,
             CandidateCellForSweepingRequest request);
 
+
```

Comment:
There must be balance. If you fix whitespace in one place, you must break it in another.